.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please add this folder .vscode/ to .gitignore

Reply:
This resolved